moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -147,6 +140,11 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       label: "Status",
```

Comment:
i feel like we only need is and is not here as operators? and maybe is blank if its possible to have a blank status?

Reply:
I'm going to see if we have an issue related to this or create one if not. I agree that we could sweep through the searchable fields and take another look at whether they make sense! 🧹